CHANGELOG.md

```diff
@@ -16,9 +16,16 @@ All notable changes to the Zowe Client Python SDK will be documented in this fil
 
 ## `1.0.0-dev21`
 
+### Enhancements
+- Split response classes into individual modules for better maintainability:
+    - `JobResponse` now lives in `job_response.py`
+    - `SpoolResponse` now lives in `spool_response.py`
+    - `StatusResponse` now lives in `status_response.py`
+- Updated imports in relevant modules to reflect these changes.
```

Comment:
These enhancements should be moved under a `## Recent Changes` header at the top of the CHANGELOG.md file.